cmd/swapd/main.go

```diff
@@ -213,8 +215,9 @@ type xmrmakerHandler interface {
 }
 
 type daemon struct {
-	ctx    context.Context
-	cancel context.CancelFunc
+	ctx      context.Context
+	cancel   context.CancelFunc
+	database *db.Database
```

Comment:
We are not in a package named `swap` or `offer`. I think it would help the reader understand the code if we are more specific in our naming like calling the type `swapDaemon` or `swapd` and the database `offerDatabase` or `offerDB`.

Reply:
I plan to put other things in the database other than just offers (see #83) which is why I just named it `database`. I can rename `daemon` though